cdap-client/src/main/java/co/cask/cdap/client/MetricsClient.java

```diff
@@ -59,6 +63,62 @@ public MetricsClient(ClientConfig config) {
   }
 
   /**
+   * Searches for metrics matching the given tags.
+   *
+   * @param tags the tags to match
+   * @return the metrics matching the given tags
+   * @throws IOException if a network error occurred
+   * @throws UnauthorizedException if the request is not authorized successfully in the gateway server
+   */
+  public List<MetricTagValue> search(List<String> tags)
+    throws IOException, UnauthorizedException {
+
+    List<String> queryParts = Lists.newArrayList();
+    queryParts.add("target=tag");
```

Comment:
the javadoc says searches for metrics matching the tags, while here the target is tag, should fix the doc and also add searchMetrics method as well.